src/ocean/text/Arguments.d

```diff
@@ -1293,8 +1293,10 @@ public class Arguments
         }
 
         T num;
-        enforce(toInteger(value, num));
-        return num;
+        if (toInteger(value, num))
```

Comment:
Actually, probably also worth mentioning this behaviour in the method doc. (The lack of doc is probably partly what caused this regression in the first place.)

Reply:
Updated